rspace/src/main/scala/coop/rchain/rspace/RSpace.scala

```diff
@@ -461,7 +464,8 @@ object RSpace {
       contextShift: ContextShift[F],
       scheduler: ExecutionContext,
       metricsF: Metrics[F],
-      spanF: Span[F]
+      spanF: Span[F],
+      par: Par[F]
```

Comment:
I think the `Parallel` typeclass should be used here (and other places) instead of `Par`

Reply:
could you share a snippet on how to pass `Parallel` around?

Reply:
Just like every other typeclass. `implicit parallel: cats.Parallel[M, F],`

Reply:
that would require the awkwardness of yet another type parameter. The Par[F] typeclass is there precisely to avoid that.  Notice also that cats 3 is going to incorporate the Par[F] approach.

Reply:
Fair enough